WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/PaymentCaptureOrchestratorTests.swift

```diff
@@ -0,0 +1,225 @@
+import XCTest
+import Yosemite
+import Hardware
+@testable import WooCommerce
+
+final class PaymentCaptureOrchestratorTests: XCTestCase {
+
+    private var stores: MockStoresManager!
+    private var sessionManager: SessionManager!
+    private var sut: PaymentCaptureOrchestrator!
+    private let sampleSiteID: Int64 = 1234
+
+    override func setUp() {
+        super.setUp()
+        sessionManager = SessionManager.makeForTesting(defaultSite: .fake().copy(
+            siteID: sampleSiteID,
+            name: "AwesomeStore"))
+        stores = MockStoresManager(sessionManager: sessionManager)
+        sut = PaymentCaptureOrchestrator(stores: stores,
+                                         paymentReceiptEmailParameterDeterminer: MockReceiptEmailParameterDeterminer(),
+                                         celebration: MockPaymentCaptureCelebration())
```

Comment:
TIL: I was wondering why we need to declare a protocol and inject a mock here, and is so we could get rid of the cha-ching sound in unit testing :D

Reply:
Yep! I almost missed this and was glad I had sound on, otherwise I'd have had everyone cursing me for re-introducing the really loud noise whenever you run tests!